packages/shared/src/lib/core/nfts/types/nft-metadata.type.ts

```diff
@@ -0,0 +1,3 @@
+import { IErc721Metadata, IIrc27Metadata } from '../interfaces'
+
+export type NftMetadata = IIrc27Metadata | IErc721Metadata
```

Comment:
This will add confusion, best to have an nft union type and then in each nft interface we should know the exact nft metadata interface we are working with